src/style.css

```
@@ -0,0 +1,63 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir.ttc);
+}
+
+.search__bar {
+  display: flex;
+  box-sizing: border-box;
+  height: 70px;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  margin: 20px 0;
+}
+
+.search__label {
+  flex-shrink: 0;
+  flex-basis: 19px;
+  margin-left: 26px;
+  background: url(./images/Search.svg) no-repeat center/contain;
+}
+
+.search__input {
+  width: 100%;
+  border: none;
+  padding-left: 17px;
+  font-family: "Avenir", sans-serif;
```

Move `font-family: "Avenir", sans-serif;` into `html` selector. For input use `inherit` value
```suggestion
  font-family: inherit;
```